test/operator/test_operator.py

```diff
@@ -110,6 +110,22 @@ def _loc(*args):
     weights=(0.5 + 0.3j, 0.5 - 0.3j),  # must add h.c.
 )
 
+operators[
+    "FermionOperator2ndJax(_mode=default-scan)"
+] = nkx.operator.FermionOperator2ndJax(
+    hi,
+    terms=(((0, 1), (3, 0)), ((3, 1), (0, 0))),
+    weights=(0.5 + 0.3j, 0.5 - 0.3j),  # must add h.c.
+)
+
+# TODO: @inailuig Fix this broken test
+# operators["FermionOperator2ndJax(_mode=mask)"] = nkx.operator.FermionOperator2ndJax(
+#    hi,
+#    terms=(((0, 1), (3, 0)), ((3, 1), (0, 0))),
+#    weights=(0.5 + 0.3j, 0.5 - 0.3j),  # must add h.c.
+#    _mode="mask"
+# )
```

Comment:
@inailuig the mode mask is broken. I commented the test, but if at some point you have some time it would be lovely if you could fix this.

Regardless I'll merge this as is.